harveyaot_AlphaTaiBai_demo_app_100.py

```
@@ -0,0 +1,74 @@
+#coding:utf-8
+import os
+from flask import Flask, render_template, request, send_from_directory
+from label_image import *
```

Ruff (0.13.1)

`from label_image import *` used; unable to detect undefined names

(F403)